whylog/constraints/__init__.py

```diff
@@ -126,12 +130,32 @@ def verify(self, group_contents, param_dict):
 
         pass
 
+    def _convert_params_values(self):
+        problems = []
+        for param_name, converter_type in six.iteritems(self.PARAMS):
+            param_val = self.params.get(param_name)
+            if param_val is not None:
+                converter = get_converter(converter_type)
+                try:
+                    converted_val = converter.convert(param_val)
+                except ValueError:
```

Comment:
I'm not sure that ValueError is the only one that you should catch here. Try to convince me.